draft-simmen-rpp-dns-data-01.md

```diff
@@ -357,13 +357,6 @@ DNS configuration of Host Object is specified by NS, A and AAAA configuration wi
       "@type": "Host",
       "name": "ns.example.com",
       "dns": [
-        {
-          "name": "@",
-          "type": "ns",
-          "rdata": {
-            "nsdname": "ns.example.com"
```

Comment:
@christian-simmen not 100% sure about that.
You mean it's not needed because redundant to the name above? I would still like to have NS entry explicit.
How would you model a future Host object with DELEG and without NS?

Reply:
As talked, depending on linking structure both ways are possible. Having it in the host object will give more flexibility. will revert it